Containerfile

```diff
@@ -45,6 +45,9 @@ RUN export LD_LIBRARY_PATH=/usr/local/cuda-12/compat:$LD_LIBRARY_PATH; \
             -i ocp-product-docs-$(echo $OCP_VERSION | sed 's/\./_/g') -v ${OCP_VERSION} -hb $HERMETIC; \
     done
 
+RUN LATEST_VERSION=$(ls -1 vector_db/ocp_product_docs/ | sort -V | tail -n 1) && \
+    ln -s vector_db/ocp_product_docs/${LATEST_VERSION} vector_db/ocp_product_docs/default
```

Comment:
@raptorsun a couple of things:
- would it be clearer to name the link `latest`?
- the code should look like
```
cd vector_db/ocp_product_docs && ln -s ${LATEST_VERSION} latest
```
or we're going to end up with a dangling link:
```
$ ln -s vector_db/ocp_product_docs/4.15 vector_db/ocp_product_docs/default
$ ls -l vector_db/ocp_product_docs/
total 4
drwxr-xr-x. 1 joe joe 216 Jun 10  2024 4.15
lrwxrwxrwx. 1 joe joe  31 May 13 20:10 default -> vector_db/ocp_product_docs/4.15
$
```